duckduckgo/iOS
# 14. Implemented Error handlers for all subscription scenarios

Task/Issue URL: https://app.asana.com/0/414235014887631/1206707680638882/f

**Description**:
This adds different error handling 'handles' to attach pixels and events, and streamlines the overall error handling flow in the UI

**Steps to test this PR**:
Confirm the 'happy paths' work as expected..

For example:
1. Get a subscrioption
2. Add an email
3. Remove from device
4. Restore Via App Store connect
5. Remove from device
6. Restore via email

<!--
---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
